src/gui_common/charts/DataPoint.cs

```diff
@@ -94,6 +96,9 @@ public override void _Ready()
         graphMarkerCircle = GD.Load<Texture>("res://assets/textures/gui/bevel/graphMarkerCircle.png");
         graphMarkerCross = GD.Load<Texture>("res://assets/textures/gui/bevel/graphMarkerCross.png");
 
+        // Used an old icon here
+        graphMarkerSkull = GD.Load<Texture>("res://assets/textures/gui/bevel/SuicideIcon.png");
+
```

Comment:
```suggestion
        graphMarkerSkull = GD.Load<Texture>("res://assets/textures/gui/bevel/SuicideIcon.png");

```

If we don't want to use the old icon, we'll decide that here and change it before the merge. If we're cool with it, this will just be another icon. Either way, separating it and noting it here will only confuse the future dev.